cds_migrator_kit/rdm/records/transform/xml_processing/rules/it_meetings.py

```diff
@@ -0,0 +1,321 @@
+import os
+import pickle
+from datetime import datetime
+from urllib.parse import ParseResult, parse_qs, urlparse, urlunparse
+
+from dateutil.parser import ParserError, parse
+from dojson.errors import IgnoreKey
+
+from cds_migrator_kit.errors import UnexpectedValue
+from cds_migrator_kit.transform.xml_processing.quality.decorators import (
+    for_each_value,
+    require,
+)
+from cds_migrator_kit.transform.xml_processing.quality.parsers import (
+    StringValue,
+    clean_val,
+)
+from cds_migrator_kit.videos.weblecture_migration.transform.xml_processing.quality.identifiers import (
+    get_new_indico_id,
+)
+
+from ...models.it_meetings import it_meetings_model as model
+
+
+@model.over("resource_type", "^980__", override=True)
+def resource_type(self, key, value):
+    """Translates resource_type."""
+    value = value.get("a")
+    if not value:
+        raise IgnoreKey("resource_type")
+    value = value.lower()
+
+    map = {
+        "contributionsfromindico": {"id": "presentation"},
+        "eventsfromindico": {"id": "event"},
+    }
+    try:
+        return map[value]
+    except KeyError:
+        raise UnexpectedValue(
+            "Unknown resource type (IT Meetings)", field=key, value=value
+        )
+
+
+@model.over("collection", "^690C_")
+@for_each_value
+def collection(self, key, value):
+    """Translates collection field."""
+    collection = value.get("a").strip().lower()
+    if collection not in ["conference", "announcement"]:
+        raise UnexpectedValue(subfield="a", field=key, value=value)
+    subjects = self.get("subjects", [])
+    subjects.append(
+        {
+            "subject": collection.upper(),
```

Comment:
can you add a comment in which cases this was kicking in? because maybe we need to use a special keyword to store this info

Reply:
yes, I meant though a comment in the code for us to remember why this was implemented